neo/Implementations/Wallets/peewee/UserWallet.py

```diff
@@ -328,9 +327,12 @@ def LoadNEP5Tokens(self):
     def LoadStoredData(self, key):
         logger.debug("Looking for key %s " % key)
         try:
-            return Key.get(Name=key).Value
+            return Key.get(Name=key).Value.decode('utf-8')
```

Comment:
Can you please elaborate why this addition is needed? I compared the output of `.Value` in peewee 2.x and 3.x and it always returns `bytes`, so I'm wondering why this is now needed.

Reply:
Yes, the problem is the inherent differences between peewee 3 and peewee 2. In peewee 2, the code operated like this when looking at a _CharField --> _StringField
def coerce_to_unicode(s, encoding='utf-8'):
    if isinstance(s, unicode_type):
        return s
    elif isinstance(s, string_type):
        try:
            return s.decode(encoding)
        except UnicodeDecodeError:
            return s
    return unicode_type(s)

So, it would automatically try to decode it into a string. If it encountered an exception, it would just return the bytes. So, I needed to be able to mimic this operation to be able to get our code to work. Specifically, if you don't try to decode the bytes first, you will receive a migration error as I referenced [here](https://github.com/CityOfZion/neo-python/pull/524#issuecomment-408403892) and then resolved it as mentioned [here](https://github.com/CityOfZion/neo-python/pull/524#issuecomment-408695613)